lib/Timer/Timer.test.js

```diff
@@ -165,4 +184,15 @@ describe('Timer', () => {
       })
     })
   })
+  describe('resetTimer', () => {
+    it('Should clear associated interval', () => {
```

Comment:
Should we also have a test that assert that `startTime` and `endTime` get cleared?